docs/command-line-usage.md

```diff
@@ -60,6 +60,7 @@ Options:
                                  # Default: 98
   [--license=LICENSE]            # The SPDX ID or key of the license to compare (implies --diff)
   [--diff], [--no-diff]          # Compare the license to the closest match
+  [--ref=REF]                    # The name of the commit/branch/tag to search
```

Comment:
Should this and the detect.rb option specify github.com since it'll only work with repos there with this implementation? IIUC the option will silently not have any effect if executed against a local repo, for instance, right?

Reply:
Ah yeah, I'd just pulled the default `help detect` output. I added a note that ref only works for github PATHs.